packages/circuit-ui/components/Grid/Grid/__snapshots__/Grid.spec.tsx.snap

```diff
@@ -4,6 +4,9 @@ exports[`Grid should render with default styles 1`] = `
 .circuit-0 {
   margin: 0 auto;
   width: 100%;
+  max-width: 880px;
+  padding-left: calc(16px / 2);
+  padding-right: calc(16px / 2);
```

Comment:
This addition to the default styles is caused by the consistent use of `composeBreakpoints` in all grid components. It has no effect since it's overridden by the breakpoint styles below which have higher specificity.